numba_dppy/dppy_passbuilder.py

```diff
@@ -2,30 +2,49 @@
 
```

Comment:
Was this file reformatted using black? We should avoid mixing reformatting with logical changes. Reformatting should be a separate PR. Otherwise it creates too much noise.